git/git.go

```diff
@@ -164,8 +165,10 @@ func CommitBody(sha string) (string, error) {
 // Push publishes a git ref to a remote and sets up upstream configuration
 func Push(remote string, ref string) error {
 	pushCmd := GitCommand("push", "--set-upstream", remote, ref)
+	regexp := regexp.MustCompile("^remote:.*$")
```

Comment:
This will completely remove any lines that start with `remote:`. That seems aggressive, thoughts on making this a more specific regex?

Reply:
That's definitely aggressive, since we want to allow the remote to keep printing other messages. Instead of filtering by a single regex, could this filter the `Create a pull request for '.*' on GitHub by visiting:` line and the line that immediately follows it, since we know that the following line will be a URL?

That will leave two `remote:` blank lines, which you could also omit from output if they touch the omitted lines as described above.